src/lib/routes/admin-api/context.ts

```diff
@@ -243,4 +262,31 @@ export class ContextController extends Controller {
         await this.contextService.validateName(name);
         res.status(200).end();
     }
+
+    async getStrategiesByContextField(
+        req: IAuthRequest<{ contextField: string }>,
+        res: Response,
+    ): Promise<void> {
+        const { contextField } = req.params;
+        const contextFields =
+            await this.contextService.getStrategiesByContextField(contextField);
+
+        // Remove unnecessary IFeatureStrategy fields from the response.
+        const contextFieldStrategies = contextFields.map((strategy) => ({
```

Comment:
why does service return data that we don't need and forces this mapping? if the service used anywhere else?
Could a service return a fit for purpose read model instead?